tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -94,6 +87,13 @@ private SpanContext(ulong? traceId, string serviceName)
             ServiceName = serviceName;
         }
 
+        /// <summary>
+        /// Gets a <see cref="ISpanContext"/> with default values. Can be used as the value for
+        /// <see cref="SpanCreationSettings.Parent"/> in <see cref="Tracer.StartActive(string, SpanCreationSettings)"/>
+        /// to specify that the new span should not inherit the currently active scope as its parent.
+        /// </summary>
+        public static ISpanContext None { get; } = new ReadOnlySpanContext(traceId: 0, spanId: 0, serviceName: null);
```

Comment:
Any particular reason for this change? Field seems pretty standard for this e.g. `string.Empty`?

Reply:
I started because it generally feels weird to me to have public fields in a class. Encapsulation and all that. Although I confess that I looked at `DateTime.Now` as an example yesterday, but just now realized a property makes sense there because the value changes over time. 🤦🏽‍♂️

I'll revert the code changes and leave only the comments. Thanks!